browser/containers/Group.tsx

```diff
@@ -111,25 +111,69 @@ export default class Group extends React.Component<Props, State> {
     this.setState({
       alert: (
         <SweetAlert
-          type="input"
-          showCancel={true}
-          inputType="month"
-          title="Report Month"
+          title="Report Date"
           onConfirm={this.downloadReport}
-          onCancel={this.hideAlert}
-          cancelBtnBsStyle="warning"
+          showConfirm={false}
         >
           Please select the specific month and year for the report.
+          <form onSubmit={this.downloadReport}>
```

Comment:
Since this is being used in more than one location why not factor it out into a stand alone component?